include/tscore/ink_config.h.cmake.in

```diff
@@ -171,3 +171,4 @@ const int DEFAULT_STACKSIZE = @DEFAULT_STACK_SIZE@;
 #cmakedefine YAMLCPP_LIB_VERSION "@YAMLCPP_LIB_VERSION@"
 
 #cmakedefine01 TS_HAS_CRIPTS
+#cmakedefine SRC_BUILD_DIR "@SRC_BUILD_DIR@"
```

Comment:
Since this is just for a unit test, you can use:
```
add_compile_definitions(SRC_BUILD_DIR="${CMAKE_BINARY_DIR}")
```

in the `src/proxy/http/unit-tests/CMakeLists.txt`